bundles/io/org.eclipse.smarthome.io.transport.mdns/src/main/java/org/eclipse/smarthome/io/transport/mdns/discovery/MDNSDiscoveryService.java

```diff
@@ -62,12 +62,6 @@ public void unsetMDNSClient(MDNSClient mdnsClient) {
 
     @Override
     protected void startBackgroundDiscovery() {
-        scheduler.schedule(new Runnable() {
```

Comment:
After several discussions I would agree. Added startScan() to startBackgroundDiscovery().